cx_Freeze/macdist.py

```diff
@@ -157,7 +158,7 @@ class bdist_mac(Command):
         (
             "rpath-lib-folder",
             None,
-            "replace @rpath with given folder for any files",
+            "[DEPRECIATED]",
```

Comment:
I usually see in similar situations, something like: OBSOLETE, DISCONTINUED or DEPRECATED.

Reply:
Sorry, typo.  I deal with "depreciation" a lot in my job day-to-day, so wrote that without thinking.

Reply:
I meant, without changing what you've already done,
```
class bdist_mac(Command):
    plist_items: List[...]
```

Reply:
Sorry--missed that comment. Would be happy to remove the draft.

(Did you intend to close this pull request unmerged?)

Reply:
I didn't close it, but I can see that you changed the base branch to develop, and when updating my develop, it is recreated, so it closed what was pending ....

Reply:
Ah, got it.  Do you prefer the pull requests on master?

Reply:
It is not a matter of my preference. The develop branch is unstable.